conan/tools/cmake/cmakedeps/templates/target_configuration.py

```diff
@@ -88,7 +88,6 @@ def template(self):
 
         # FIXME: What is the result of this for multi-config? All configs adding themselves to path?
         set(CMAKE_MODULE_PATH {{ '${' }}{{ pkg_name }}_BUILD_DIRS{{ config_suffix }}} {{ '${' }}CMAKE_MODULE_PATH})
```

Comment:
this one is harmless but honestly a config file manipulating CMAKE_MODULE_PATH is unusual, I think it's too intrusive.